web/src/lib/domain-logic/getNavBarBusinessTitle.ts

```diff
@@ -52,7 +52,7 @@ export const getNavBarBusinessTitle = (
     }
   }
 
-  if (businessPersona === "FOREIGN" && isRemoteWorkerOrSeller(business)) {
```

Comment:
We can abstract away logic like this or situations where the business is undefined.